Solon_Burleson/ForLoopBasicsII.py

```diff
@@ -0,0 +1,120 @@
+# Biggie Size
+
+def makeitBig(list):
+    for x in range(len(list)):
+        if list[x] > 0:
+            list[x] = "big"
+    return list
+
+print(makeitBig([-1,3,5,-5]))
+
+# Count Positives
+
+def countPositives(list):
+    count = 0
+    for x in range(len(list)):
+        if list[x] > 0:
+            count += 1
+    list[len(list)-1] = count
+    return list
+
+print(countPositives([-1,1,1,1]))
+
+# Sum Total
+
+def sumTotal(list):
+    sum = 0
+    for x in range(len(list)):
+        sum += list[x]
+    return sum
+
+print(sumTotal([1,2,3,4]))
+
+# Average
+
+def average(list):
+    sum = 0
+    avg = 0
+    for x in range(len(list)):
+        sum += list[x]
+    avg = sum / len(list)
+    return avg
+
+print(average([1,2,3,4]))
+
+# Length
+
+def length(list):
+    length = 0
+    for x in range(len(list)):
+        length+=1
+    return length
+
+print(length([1,2,3,4]))
+
+# Minimum
+
+def minimum(list):
+    min = list[0]
```

Comment:
if the list is empty then you'll get an error. Assigning the min value in the else would solve it.